shell/renderer/api/electron_api_ipc_renderer.cc

```diff
@@ -218,13 +230,14 @@ class IPCRenderer : public gin::Wrappable<IPCRenderer>,
     }
 
     blink::CloneableMessage result;
-    electron_browser_ptr_->MessageSync(internal, channel, std::move(message),
-                                       &result);
+    electron_browser->MessageSync(internal, channel, std::move(message),
+                                  &result);
     return electron::DeserializeV8Value(isolate, result);
   }
 
   v8::Global<v8::Context> weak_context_;
-  electron::mojom::ElectronBrowserPtr electron_browser_ptr_;
+  mojo::PendingRemote<electron::mojom::ElectronBrowser>
```

Comment:
`PendingRemote` -> `Remote` and revert everywhere to use `electron_browser_remote_` directly without a local `mojo::Remote` conversion.